WordPress/Classes/Models/Notifications/Notification.h

```diff
@@ -139,6 +141,7 @@ typedef NS_ENUM(NSInteger, NoteBlockGroupType)
 @property (nonatomic, strong, readwrite) NSString           *textOverride;
 
 - (NotificationRange *)notificationRangeWithUrl:(NSURL *)url;
+- (NotificationRange *)notificationRangeWithCommentId:(NSNumber *)commentId;
```

Comment:
Since we're adding this method now, and it seems like a good candidate, let's add a unit test for it.

Also let's document it - eventually we should have all of this class documented, but let's focus on starting with the new method for now.